src/styles/index.ts

```diff
@@ -1247,8 +1247,6 @@ const styles = (theme: ThemeColors) =>
         },
 
         textInputAndIconContainer: {
```

Comment:
this style is only used inside `TextInput` and the changes were tested as part of this task - it does not affect anything else